state/action.go

```diff
@@ -682,8 +682,11 @@ func (m *Model) EnqueueAction(operationID string, receiver names.Tag,
 		return nil, errors.New("action name required")
 	}
 
+	checkNotDead := true
 	receiverCollectionName, receiverId, err := m.st.tagToCollectionAndId(receiver)
-	if err != nil {
+	if errors.Is(err, errors.NotImplemented) {
+		checkNotDead = false
```

Comment:
I'm guessing the receiver would be a user.

Reply:
Makes sense, in your opinion is it worth just removing this entity not dead check entirely rather than having this conditional logic?